Makefile

```diff
@@ -24,18 +24,25 @@ devtools:
 	go install github.com/bufbuild/buf/cmd/buf@v1.25.0
 	go install mvdan.cc/gofumpt@latest
 	go install github.com/rakyll/statik@v0.1
+	go install github.com/pacviewer/jrpc-gateway/protoc-gen-jrpc-gateway@v0.1.0
 
 ########################################
 ### Building
-build:
-	go build -o ./build/pactus-daemon$(EXE) ./cmd/daemon
-	go build -o ./build/pactus-wallet$(EXE) ./cmd/wallet
-	go build -o ./build/pactus-shell$(EXE) 	./cmd/shell
+build: build_daemon build_wallet build_shell
 
 build_race:
 	go build -race -o ./build/pactus-daemon$(EXE) ./cmd/daemon
 	go build -race -o ./build/pactus-wallet$(EXE) ./cmd/wallet
 
+build_daemon:
```

Comment:
Explain why did you create new target per binary?

Reply:
I wanted to build daemon standalone and test it but there wasn't target for it.

Reply:
Let's build all-in-on as i was before.

Reply:
sure